spec/workers/slack_id_import_worker_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
+
+RSpec.describe SlackIdImportWorker, type: :worker do
+  let(:time) { (Time.zone.today + 1.hours).to_datetime }
+  let(:scheduled_job) { described_class.perform_at(time, 'Awesome', true) }
+
+  context 'occurs weekly' do
+    it 'occurs at expected time' do
+      scheduled_job
+      assert_equal true, described_class.jobs.last['jid'].include?(scheduled_job)
+      expect(described_class).to have_enqueued_sidekiq_job('Awesome', true)
```

Comment:
Your spec says `occurs at expected time`, so I'd probably check if it _really_ does occur at the expected time, not just that it _occurs_. WDYTA adding `.at(time)` on LOC 14 (https://github.com/philostler/rspec-sidekiq#have_enqueued_sidekiq_job)?

Reply:
True, spec says "occurs at expected time", but test is checking something else - if the worker is scheduled. Will fix that